project.php

```diff
@@ -1654,6 +1654,10 @@ function echo_download_zip($link_text, $discriminator)
     echo "</a>";
     echo_byte_size($filesize_b);
     echo_last_modified($last_modified);
+    // output a warning if the file was generated before the UTF-8 cutover
+    if ($discriminator == "" && $last_modified && $last_modified < 1589698800) {
+        echo "<br><span class='warning'>" . _("This file was generated before the Unicode conversion. If you are just starting to PP this project, it's recommended you regenerate the file.") . "</span>";
```

Comment:
I recommend that
> it's recommended you regenerate the file.

be changed to
> please ask a site administrator to regenerate the file.

unless you want to get wordy and point out that they can generate a CTF for download that ("see 'Download Concatenated Text' below"), while it won't include GWL/BWL, will have a regenerated version of the text. Only site admins can regenerate the post files that are stored on the server.

Reply:
Wording updated, thanks for the assist!